nginx-fem-redirects.conf

```
@@ -31,135 +31,135 @@ location ~* ^/about/(?:team|publications)/?$ {
 }
 
 # FEM projects
-location ~* ^/projects/zookeeper/galaxy-zoo-weird-and-wonderful/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/(?:[\w-]*?/)?zookeeper/galaxy-zoo-weird-and-wonderful/?(?:(classify|about)(?:/.+?)?)?/?$ {
     resolver 1.1.1.1;
     proxy_pass $fe_project_uri;
     proxy_set_header Host $fe_project_host;
 
     include /etc/nginx/proxy-security-headers.conf;
 }
 
-location ~* ^/projects/hughdickinson/superwasp-black-hole-hunters/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/(?:[\w-]*?/)?hughdickinson/superwasp-black-hole-hunters/?(?:(classify|about)(?:/.+?)?)?/?$ {
     resolver 1.1.1.1;
     proxy_pass $fe_project_uri;
     proxy_set_header Host $fe_project_host;
 
     include /etc/nginx/proxy-security-headers.conf;
 }
 
-location ~* ^/projects/bogden/scarlets-and-blues/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/(?:[\w-]*?/)?bogden/scarlets-and-blues/?(?:(classify|about)(?:/.+?)?)?/?$ {
     resolver 1.1.1.1;
     proxy_pass $fe_project_uri;
     proxy_set_header Host $fe_project_host;
 
     include /etc/nginx/proxy-security-headers.conf;
 }
 
-location ~* ^/projects/kmc35/peoples-contest-digital-archive/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/(?:[\w-]*?/)?kmc35/peoples-contest-digital-archive/?(?:(classify|about)(?:/.+?)?)?/?$ {
     resolver 1.1.1.1;
     proxy_pass $fe_project_uri;
     proxy_set_header Host $fe_project_host;
 
     include /etc/nginx/proxy-security-headers.conf;
 }
 
-location ~* ^/projects/adamamiller/zwickys-stellar-sleuths/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/(?:[\w-]*?/)?adamamiller/zwickys-stellar-sleuths/?(?:(classify|about)(?:/.+?)?)?/?$ {
     resolver 1.1.1.1;
     proxy_pass $fe_project_uri;
     proxy_set_header Host $fe_project_host;
 
     include /etc/nginx/proxy-security-headers.conf;
 }
 
-location ~* ^/projects/humphrydavy/davy-notebooks-project/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/(?:[\w-]*?/)?humphrydavy/davy-notebooks-project/?(?:(classify|about)(?:/.+?)?)?/?$ {
     resolver 1.1.1.1;
     proxy_pass $fe_project_uri;
     proxy_set_header Host $fe_project_host;
 
     include /etc/nginx/proxy-security-headers.conf;
 }
 
-location ~* ^/projects/mainehistory/beyond-borders-transcribing-historic-maine-land-documents/?(?:(classify|about)(?:/.+?)?)?/?$ {
+location ~* ^/projects/mainehistory/(?:[\w-]*?/)?beyond-borders-transcribing-historic-maine-land-documents/?(?:(classify|about)(?:/.+?)?)?/?$ {
```

this one was switched around 
```suggestion
location ~* ^/projects/(?:[\w-]*?/)?mainehistory/beyond-borders-transcribing-historic-maine-land-documents/?(?:(classify|about)(?:/.+?)?)?/?$ {```

---

Good eye!